misc/python/materialize/version_list.py

```diff
@@ -270,6 +271,12 @@ def get_published_minor_mz_versions(
     all_versions = get_all_mz_versions(newest_first=True)
     minor_versions: dict[str, MzVersion] = {}
 
+    if exclude_current_minor:
+        version = MzVersion.parse_cargo()
+        current_version = f"{version.major}.{version.minor}"
+    else:
+        current_version = None
```

Comment:
Can we simplify the code by removing the if-else and just always create the `current_minor_version`?